.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,11 +26,9 @@ This PR closes [issue number] and relates to [issue or PR number].
 **Further comments**  
 <!-- Specify questions or related information -->
 
-**Checklist**  
-<!-- Please delete options that are not relevant -->
+**Definition of Done checklist**  
+- [ ] My code meets the Acceptance Criteria
 - [ ] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
+- [ ] I have performed a self-review of my own code and commented any hard-to-understand areas
```

Comment:
What's the reasoning behind joining these two criteria (self-review and comments)?

Reply:
I felt like they were related + slightly more time efficient to only mark one box instead of two :nerd_face: But I'm up for separating them if you feel like that is better

Reply:
I like the time efficiency aspect :smile:! I'm fine with keeping them as you put them here (although I would say there not totally related?). As long as they're there and needs to be checked, I think it's all good!